satellite/console/consoleweb/consoleapi/auth.go

```diff
@@ -0,0 +1,299 @@
+// Copyright (C) 2019 Storj Labs, Inc.
+// See LICENSE for copying information.
+
+package consoleapi
+
+import (
+	"context"
+	"encoding/json"
+	"net/http"
+	"strings"
+
+	"github.com/gorilla/mux"
+	"github.com/skyrings/skyring-common/tools/uuid"
+	"github.com/zeebo/errs"
+
+	"go.uber.org/zap"
+
+	"storj.io/storj/internal/post"
+	"storj.io/storj/pkg/auth"
+	"storj.io/storj/satellite/console"
+	"storj.io/storj/satellite/console/consoleweb/consoleql"
+	"storj.io/storj/satellite/mailservice"
+)
+
+var Error = errs.Class("satellite console auth api error")
```

Comment:
we use named logger, so lets leave only "console auth api error" not to duplicate text